client/my-sites/marketing/buttons/options.jsx

```diff
@@ -207,7 +207,7 @@ class SharingButtonsOptions extends Component {
 		return (
 			<fieldset className="sharing-buttons__fieldset">
 				<legend className="sharing-buttons__fieldset-heading">
-					{ translate( 'Show sharing buttons on', {
+					{ translate( 'Show like and sharing buttons on', {
```

Comment:
:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?